h5py_wrapper/wrapper.py

```diff
@@ -282,9 +284,12 @@ def _evaluate_key(f):
     Evaluate the key of f and handle non-string data types.
     """
     name = os.path.basename(f.name)  # to return only name of this level
-    if ('_key_type' in f.attrs and
-            f.attrs['_key_type'] not in ['str', 'unicode', 'string_']):
-        name = ast.literal_eval(name)
+    if '_key_type' in f.attrs:
```

Comment:
Hmm, I would say not because above, there should be a value type stored, otherwise an error is raised. Here, it is fine if there is no key type stored.